shopinvader_restapi/services/abstract_sale.py

```diff
@@ -43,27 +43,14 @@ def _convert_step(self, sale):
     def _is_item(self, line):
         return True
 
+    def _get_product_information(self, line):
+        return {"id": line.product_id.id}
```

Comment:
Before, due to the shopinvader product variant, we got all info about a product inside the SO data (product name, prices, + various custom fields...). Now I see that we just get the product ID. Is it wanted? What are the benefits?